doc/lua_api.md

```diff
@@ -8736,6 +8736,20 @@ Used by `minetest.register_node`, `minetest.register_craftitem`, and
     -- Otherwise should be name of node which the client immediately places
     -- upon digging. Server will always update with actual result shortly.
 
+    touch_controls = {
```

Comment:
alternative name suggestion that you can ignore: `touch_interaction`

Reply:
I like this one.